egs/callhome_diarization/v1/diarization/vad_to_segments.sh

```diff
@@ -26,6 +27,7 @@ if [ $# -ne 2 ]; then
   echo "    --stage (0|1)  # start script from part-way through"
   echo "    --cmd (run.pl|queue.pl...)  # specify how to run the sub-processes"
   echo "    --segmentation-opts '--opt1 opt1val --opt2 opt2val' # options for segmentation.pl"
+  echo "    --min-duration <m> # filtering out any generated subsegment with lower duration
```

Comment:
I'd change the comment to:
`# min duration in seconds for segments (smaller ones are discarded)`